docs/next/guides/building-and-testing/modules.md

```diff
@@ -481,352 +275,822 @@ new Handsontable(container, {});
+```
+
+#### Step 3: Import the CSS
+
+Handsontable's CSS file is not modularized, so you need to import it:
+
+```js
+import Handsontable from 'handsontable/base';
+
+// import Handsontable's CSS
+import 'handsontable/dist/handsontable.full.css';
+```
+
+Now, you're ready to use [any optional modules](#list-of-all-modules) of your choice.
+
+### Using editor modules
+
+Each editor module contains a different type of [cell editor](@/guides/cell-functions/cell-editor.md):
 
-Start with importing the base, `PasswordEditor` and the `registerEditor` method.
+| Module               | Cell editor alias |
+| -------------------- | ----------------- |
+| `AutocompleteEditor` | `autocomplete`    |
+| `BaseEditor`         | `base`            |
+| `CheckboxEditor`     | `checkbox`        |
+| `DateEditor`         | `date`            |
+| `DropdownEditor`     | `dropdown`        |
+| `HandsontableEditor` | `handsontable`    |
+| `NumericEditor`      | `numeric`         |
+| `PasswordEditor`     | `password`        |
+| `SelectEditor`       | `select`          |
+| `TextEditor`         | `text`            |
+
+To start using an editor module, follow the steps below.
+
+#### Step 1: Import the required modules
+
+First, make sure to [import the required modules](#importing-required-modules):
 
 ```js
-// remember to have the base imported
+// import the `handsontable/base` module
 import Handsontable from 'handsontable/base';
-// import the method for registering an editor
-// import the PasswordEditor
+
+// import Handsontable's CSS
+import 'handsontable/dist/handsontable.full.css';
+```
+
+#### Step 2: Import the editors' registering method
+
+Import the `registerEditor()` method that lets you register your chosen editor:
+
+```js
+import Handsontable from 'handsontable/base';
+import 'handsontable/dist/handsontable.full.css';
+
+// import the `registerEditor()` method
+import { registerEditor } from 'handsontable/editors';
+```
+
+#### Step 3: Import your editor module
+
+Import an editor module of your choice. For example:
+
+```js
+import Handsontable from 'handsontable/base';
+import 'handsontable/dist/handsontable.full.css';
+
+// import the `PasswordEditor` module
 import { registerEditor, PasswordEditor } from 'handsontable/editors';
 ```
 
-Afterwards you need to use the `registerEditor` method to register `PasswordEditor`:
+#### Step 4: Register your editor
+
+Register your editor, using the `registerEditor()` method. For example:
 
 ```js
-// register the editor before using it
+import Handsontable from 'handsontable/base';
+import 'handsontable/dist/handsontable.full.css';
+import { registerEditor, PasswordEditor } from 'handsontable/editors';
+
+// register the `password` editor
 registerEditor(PasswordEditor);
 ```
 
-Now, you can use `PasswordEditor`, the full example looks like this:
+::: tip
+#### Registering all editors at once
+
+If you import all of Handsonable's editor modules, register them all with the `registerAllEditors()` method:
```

Comment:
Why would I want to do that? I think that it might be useful temporarily to make development easier.

But this tip does not say how to import all editors. Do I need to put many lines of code to import all editors by name?

@budnix would the following work?

```js
import * from 'handsontable/editors'
import { registerAllEditors } from 'handsontable/registry';
registerAllEditors();
```

Reply:
I think that `registerAllEditors();` (and `registerAllPlugins();` etc.) functions can be useful for more lazy developers who don't want to selectively choose components or want to make sure that always there will get all cell types. That functions underneath imports and register all components. So in a case when in the new Handsontable release we add new cell type having the `registerAllCellTypes()` would provide new type automatically. 

About the example, I would change to:
```js
import { registerAllEditors } from 'handsontable/editors';

registerAllEditors();
```

Reply:
I updated the examples, and added links to the list of all modules: 4433c0f

Reply:
Aha! So the function `registerEditor` only **registers**, but the function `registerAllEditors` **imports** and registers. This is totally not obvious.

@kirszenbaum can you please make it apparent in the docs. I think that the methods `registerEditor` and `registerAllEditors` must never be mixed on the same snippet to avoid confusion.

The same applies for to other `registerAll*` functions.